apps/judicial-system/web/src/routes/Court/RestrictionRequest/Confirmation/Confirmation.tsx

```diff
@@ -77,7 +78,7 @@ export const Confirmation: React.FC = () => {
         // TODO: Handle error
       }
     } catch (e) {
-      // TODO: Handle error
+      toast.error(formatMessage(errors.requestRulingSignature))
```

Comment:
Consider removing this try / catch as the error is already handled in the useCase hook.